packages/bundler-plugin-core/src/types.ts

```diff
@@ -125,10 +125,8 @@ export interface Options {
      * The globbing patterns follow the implementation of the `glob` package. (https://www.npmjs.com/package/glob)
      *
      * Use the `debug` option to print information about which files end up being deleted.
-     *
-     * @hidden Not yet implemented.
      */
-    deleteAfterUpload?: string | string[];
+    deleteFilesAfterUpload?: string | string[];
```

Comment:
The default is `undefined` at the moment, right? wdyt about defaulting to the source maps assets glob path(s) instead?